### ISSUE TITLE
[BUG] Error message instead of downloading

### ISSUE THREAD

**Issue by ByteEater-pl** [xenova/chat-downloader#94]
## Basic information
- Program version:
- Python version: 3.9.5
- Operating system: 5.12.2-1-MANJARO #1 SMP PREEMPT Fri May 7 17:53:15 UTC 2021 x86_64 GNU/Linux

## Describe the bug
A long error message with page source is displayed.

### If running from the command line, provide the following:
1. The command used (including the verbose tag, `-v`):
```
~/.local/bin/chat_downloader https://www.youtube.com/watch?v=redacted
```
2. Output from the above command:
```
[INFO] Site: youtube.com
Traceback (most recent call last):
  File "/home/sandbox/.local/bin/chat_downloader", line 8, in <module>
    sys.exit(main())
  File "/home/sandbox/.local/lib/python3.9/site-packages/chat_downloader/cli.py", line 170, in main
    run(**args.__dict__)
  File "/home/sandbox/.local/lib/python3.9/site-packages/chat_downloader/chat_downloader.py", line 273, in run
    chat = downloader.get_chat(**chat_params)
  File "/home/sandbox/.local/lib/python3.9/site-packages/chat_downloader/chat_downloader.py", line 166, in get_chat
    info = self.sessions[site.__name__].get_chat(**params)
  File "/home/sandbox/.local/lib/python3.9/site-packages/chat_downloader/sites/youtube.py", line 1393, in get_chat
    return self.get_chat_by_video_id(match.group('id'), kwargs)
  File "/home/sandbox/.local/lib/python3.9/site-packages/chat_downloader/sites/youtube.py", line 1367, in get_chat_by_video_id
    initial_info = self._get_initial_video_info(video_id)
  File "/home/sandbox/.local/lib/python3.9/site-packages/chat_downloader/sites/youtube.py", line 906, in _get_initial_video_info
    raise ParsingError(
chat_downloader.errors.ParsingError: Unable to parse initial video data. <!doctype html><html lang="en" dir="ltr"><head><base href="https://consent.youtube.com/">
```
(truncated)

## Expected behavior
I'd like the chat (with all message types, including bans, deleted ones and messages not showing in Topchat, only in the all messages view – I hope it's the default but the documentation doesn't specify it) to be downloaded.

## Additional context/information
I ran it from a newly created Python virtual environment under a newly created user account for security reasons (I don't have the time now to carefully read the code before I run it; not an intended offense due to mistrust, and I hope none taken by you, it's just following good practices).

**Comment by ByteEater-pl** [comment]
I've just noticed:
> including the verbose tag, -v

So please see the full output with `-v` attached.
[stderr.txt](https://github.com/xenova/chat-downloader/files/6607011/stderr.txt)

**Comment by krichbanana** [comment]
Judging by the URL, this is the EU cookie consent page. See #85 and this youtube-dl commit: https://github.com/ytdl-org/youtube-dl/commit/14f29f087e6097feb46bdb84878924bc410a57eb

**Comment by xenova** [comment]
Hi there 👋 This is next on the TODO list. Will provide an update as soon as possible :)

**Comment by xenova** [comment]
This has been fixed in the latest version 😃 (v0.1.1).

Let me know if you have any other questions 👍
